example/app/config.yml

```diff
@@ -548,6 +569,14 @@ mboyd-test:
   stackName: mboyd-test
   stackNameNoDash: mboydTest
 
+  es:
```

Comment:
Is this intentional?

Reply:
I need to remove it. This was in here from when you gave me the branch :)